.github/workflows/e2e-multicluster-ci.yml

```diff
@@ -14,6 +14,9 @@ on:
     paths-ignore:
```

Comment:
if we remove the exceptions (because the tests are required either way), we might as well remove these here?

Reply:
Ah I see, multicluster is not a required test.